README.md

```diff
@@ -1,50 +1,12 @@
 # Terraform Validator
 
-This tool is used to validate terraform plans before they are applied. Validations are based on policies from the Config Validator [Policy Library](https://github.com/forseti-security/policy-library).
+Terraform Validator is a tool for validating compliance with organizational policies prior to applying a terraform plan.
+It can be used either as a standalone tool or in conjunction with [Forseti](https://forsetisecurity.org/) or other policy enforcement tooling.
+Terraform Validator relies on policies that are [compatible with Config Validator](https://github.com/forseti-security/policy-library/blob/master/docs/user_guide.md#how-to-set-up-constraints-with-policy-library). For examples, see the [Policy Library](https://github.com/forseti-security/policy-library).
 
-**Note**: Using Terraform Validator does _not_ require an active installation of Forseti. Terraform Validator is a self-contained binary.
```

Comment:
Let's please leave this notice (or something equally conspicuous). Unfortunately there's a common misconception that Terraform Validator is somehow coupled to Forseti.

Reply:
Hi Morgante - I replaced it with the line:

> [Terraform Validator] can be used either as a standalone tool or in conjunction with Forseti

Does that address your concern?

Reply:
I'd like to have an explicit note just because this is something people are confused about. We need to be crystal clear that it's not a requirement.

Reply:
I've put this note back in 5c01899